pandas/core/arrays/datetimelike.py

```diff
@@ -755,28 +760,30 @@ def isin(self, values) -> np.ndarray:
     # ------------------------------------------------------------------
     # Null Handling
 
-    def isna(self):
+    def isna(self) -> np.ndarray:
```

Comment:
we dont' have the typing ability to do `np.ndarray[bool]` yet right?

Reply:
not yet, correct

Reply:
looks like will be in numpy 1.21 https://github.com/numpy/numpy/pull/17719